pkgs/games/enigma/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, makeWrapper, pkg-config, curl, imagemagick, libpng, SDL, SDL_image, SDL_mixer, SDL_ttf, xercesc, xdg_utils, hicolor-icon-theme }:
+stdenv.mkDerivation rec {
+  pname = "enigma";
+  version = "1.21";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/enigma-game/enigma-${version}.tar.gz";
+    sha256 = "d872cf067d8eb560d3bb1cb17245814bc56ac3953ae1f12e2229c8eb6f82ce01";
+  };
+
+  # patches lifted from Arch
+  patches = [ ./src_client.cc.patch ./src_lev_Proxy.cc.patch ./src_Value.cc.patch ];
```

Comment:
Can we fetch them via fetchpatch?

Reply:
We could, but it wouldn't be fun, because the patches require different `-p0`/`-p1` settings...